### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by Marius1311** [theislab/interactive_plotting#23]
At the moment, there is absolutely no documentation on the functions. Would there by a quick and easy way to set up a documentation page on github? Like these pages in the scanpy api that just show the docstring with some explanation?

**Comment by michalk8** [comment]
Could you please elaborate on that? All these functions have docstrings, if you mean something along the lines of https://scanpy.readthedocs.io/en/latest/, I think it's too early for that. Perphaps more informative README.md will do the trick.

**Comment by Marius1311** [comment]
Yes, that's what I meant. I didn't know it would be that much work - it that case, I agree with a more elaborate readme, maybe just 2-3 bullet points per function of what it's meant for

**Comment by michalk8** [comment]
> Yes, that's what I meant. I didn't know it would be that much work - it that case, I agree with a more elaborate readme, maybe just 2-3 bullet points per function of what it's meant for

I'd err on the side of simplicity for now and just do the bullet points. I've created 2 closely related issues #27, #28, which deal with typing and unification of docstrings. Once they are finished, I will take a look at `sphinx`/other documentation generator tools.

**Comment by Marius1311** [comment]
yes, that sounds good, that's what I meant originally: some tool that automatically takes your docstrings and generates a scanpy-like API from there.

**Comment by Marius1311** [comment]
As soon as there is some sort of documentation, I think we should open an issue on scanpy to spread the work about this repo and also to discuss where we want to take this.

**Comment by michalk8** [comment]
> As soon as there is some sort of documentation, I think we should open an issue on scanpy to spread the work about this repo and also to discuss where we want to take this.

Agreed, I wanted to clean this beforehand, but I can do it in the meantime. It needs a prettier (less general) name than `interactive_plotting`- any ideas? (mine was `sciplot`, as in **SC**anpy**I**nteractive**PLOT**ing, but it's very close to `scPlot`...)

**Comment by Marius1311** [comment]
mhm. I don't at the moment - let's add Juan to the repo so he can help us brainstorm. Let's do this in a separate issue. Can you add Juan? I don't have admin rights here.
